### ISSUE TITLE
Basemap toggle button error

### ISSUE THREAD

**Issue by MarcosMoreu** [UCL/kapta-mobile#20]
It works initially, but after clicking on share image/data, the button is disabled/error

**Comment by acholyn** [comment]
I'm not having this issue, is this still the case? Is the modal closed when you try?

**Comment by MarcosMoreu** [comment]
yes. the modal is closed. Try to reproduce the issue by typing, click OK and then the basemap button is not working.

**Comment by acholyn** [comment]
Sorry, so to confirm is it after sharing data/image, then typing in the caption bar that the button doesn't work?

**Comment by acholyn** [comment]
ah, yes it is. It's due to overlapping elements
